subprojects/docs/src/docs/userguide/running-builds/performance.adoc

```diff
@@ -191,6 +191,40 @@ Some collections have dedicated types, `TaskContainer` being one of them, that h
 If you do decide to use static compilation, using an IDE can quickly show errors due to unrecognised types, properties, and methods.
 You’ll also get auto-completion, which is always handy.
 
+=== Enable the configuration cache
```

Comment:
Maybe this is something for the larger reorganization of this content, but enabling the configuration cache and the build cache are basically the _first_ things you should do. And when you enable the configuration cache, many of the other suggestions become much less important, or even irrelevant, for many teams.

So, I would present configuration cache and build cache as early as possible and encourage people to try them first, and then lead them on to other, more specific, improvements.

Also, we should mention somewhere in the doc the GE test acceleration features, test distribution and predictive test selection. These didn't exist when this content was first put together and it looks like we forget to add them.

Reply:
I plan on reorganizing and rewriting this entire page soon, so I'll put this idea on the backburner for now. When I get to that reorg/rewrite, I'll definitely focus on configuration and build caching as a first (easy!) step -- thanks for the tip.

Good point about the GE test acceleration/distribution and predictive test selection features. I would appreciate any pointers to you have to feature descriptions, points of contact, etc. so I can follow up on those and include them as I reorganize this page.

Tagging @adammurdoch so we can follow up on this.